.changeset/dry-pugs-melt.md

```diff
@@ -0,0 +1,46 @@
+---
+"@rhds/elements": minor
+---
+
+Added `<rh-tile>`.
+
+A form of selection that can be used in place of a link, checkbox, or radio button. 
+
+```html
+  <rh-tile>
+    <img slot="image" src="//fakeimg.pl/296x50" alt="296 X 50 placeholder image">
+    <div slot="title">Title</div>
+    <h2 slot="headline"><a href="#top">Link</a></h2>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+    <div slot="footer">Suspendisse eu turpis elementum</div>
+  </rh-tile>
+```
+
+Added `<rh-tile-group>`.
```

Comment:
if this doesn't have it's own top-level directory, don't include it here

if you do break this out into a top-level directly, split it into a separate changeset

if you keep it within `elements/rh-tile/rh-tile-group.js`, i suggest replacing the above snippet with this one, but keepin the prose the same.